scripts/ci_before_init_embed.bash

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-export WEBOTS_VERSION=2022a
+export WEBOTS_RELEASE_VERSION=2022a
```

Comment:
Will that work for the develop branch? The version is 2022b on the develop branch.

Reply:
We have to adapt this on `develop` branch. It should not be a problem as we will only merge `develop` into `master` once the `R2022b` will be ready to be released.